src/pages/manager-review-page/achievements/ManagerReviewAchievementsExpansionPanel.tsx

```diff
@@ -86,19 +92,19 @@ export function ManagerReviewAchievementsExpansionPanel(props: Props) {
             <ManagerReviewAchievementsRatingGroup rating="NEEDS_IMPROVEMENT" comments={projectReview.comments} />
             <ManagerReviewAchievementsRatingGroup rating={null} comments={projectReview.comments} />
           </Grid>
-          <Grid item xs>
-            <DictInputItem field={projectReview.id}>
-              <ManagerReviewEvaluationBox
-                text={i18n._('Your evaluation of {name} on project {projectName}', {
-                  name,
-                  projectName,
-                })}
-              >
-                <Rating inputLabel={i18n._('Evaluation')} type="peer" />
-                <FragmentPrompt value={rating} />
-              </ManagerReviewEvaluationBox>
-            </DictInputItem>
-          </Grid>
+          <ThoseWhoDidNotComment comments={projectReview.comments} />
+          {printing && projectReview.managerComment && (
+            <Grid item xs={12}>
+              <ManagerProjectOutput self review={projectReview.managerComment} />
+            </Grid>
+          )}
+          {!printing && (
+            <Grid item xs={12}>
+              <FormChangeDetector>
+                <ManagerReviewForm onSubmit={saveManagerProjectReview} formData={projectReview.managerComment} />
+              </FormChangeDetector>
+            </Grid>
+          )}
```

Comment:
i think use ternary operator here is better

Reply:
if I use the ternary operator, we have a two-step indentation. IMO this is more readable.